distribution/lib/Standard/Database/0.0.0-dev/src/Errors.enso

```diff
@@ -125,25 +125,15 @@ type Table_Not_Found
        - table_name: The name of the table that was not found.
        - related_query_error: The error that was thrown when the query looking
          for the table was executed.
-       - treated_as_query: Whether the table name was treated as a raw query
-         string.
-       - extra_message: An extra message to append.
-    Error (name:Text) (related_query_error:SQL_Error|Nothing) (treated_as_query:Boolean) (extra_message:Text)
+    Error (name:Text) (related_query_error:SQL_Error|Nothing)
 
     ## PRIVATE
        Pretty print the table not found error.
     to_display_text : Text
     to_display_text self =
-        base_repr = case self.treated_as_query of
-            True -> "The name " + self.name + " was treated as a query, but the query failed with the following error: " + self.related_query_error.to_display_text + "; if you want to force to use that as a table name, wrap it in `SQL_Query.Table_Name`."
-            False -> "Table " + self.name + " was not found in the database."
-        base_repr + self.extra_message
-
-    ## PRIVATE
-       Creates a copy of this error with a changed `extra_message`.
-    with_changed_extra_message : Text -> Table_Not_Found
-    with_changed_extra_message self new_extra_message =
-        Table_Not_Found.Error self.name self.related_query_error self.treated_as_query new_extra_message
+      case self.related_query_error of
+         SQL_Error -> "Table " + self.name + " was not found in the database. The query failed with the following error: " + self.related_query_error.to_display_text + "."
```

Comment:
```suggestion
         _ : SQL_Error -> "Table " + self.name + " was not found in the database. The query failed with the following error: " + self.related_query_error.to_display_text + "."
```

Is that syntax even valid?

Reply:
Ah I was writing this a bit in a hurrry.

Do we still need the `related_query_error` field here? It feels like it may become obsolete now that this is always just a 'not found'

Reply:
make_table_for_name uses the sql_error

Reply:
Hmm I'm wondering if it's needed there, but yeah let's keep it for now - it may be useful when debugging some weird issues.

Reply:
I removed it in the end. It was just added noise.